.wasser/config-ubuntu.yaml

```diff
@@ -13,15 +13,17 @@ routines:
       command: |
         cd aquarium
         ./tools/setup-dev.sh
-    - name: building image
-      command: |
-        cd aquarium
-        ./tools/build-image.sh
+        sudo adduser ubuntu kvm  
```

Comment:
So after adding this we do not need strictly run tools/build-image.sh after libvirt started?